apps/src/studio/rocketHeightLogic.js

```diff
@@ -17,10 +17,21 @@ var RocketHeightLogic = function (studio) {
   // rocket and height for use in success/failure checking
   this.rocket = null;
   this.height = 0;
+
+  // Use by successCondition/failureCondition
+  this.SECONDS_TO_RUN = 8;
 };
 RocketHeightLogic.inherits(CustomGameLogic);
 
 RocketHeightLogic.prototype.onTick = function () {
+  if (this.studio_.tickCount === 1) {
```

Comment:
It gets called after the tick is incremented (so the first tick increments 0 to 1, then calls this).